test/clj_kondo/main_test.clj

```diff
@@ -3420,6 +3420,14 @@ foo/")))
   (is (= "error" (:fail-level (#'clj-kondo.main/parse-opts ["--fail-level" "error"]))))
   (is (= "error" (:fail-level (#'clj-kondo.main/parse-opts ["--fail-level=error"])))))
 
+(deftest import-illegal-token
+  (lint! "(import \"\")")
```

Comment:
What is this expression doing?

Reply:
My bad, I was using it when testing things with CIDER, but left it by accident. I'm pushing a fix for that.